README.md

```diff
@@ -5,32 +5,33 @@
 Convert from Tensorflow to CoreML
 =================================
 
-Python package to convert from Tensorflow to CoreML format. To get the latest
-version of `tfcoreml`, please run:
+[coremltools](https://github.com/apple/coremltools) (Recommended approach)
+--------------------
+
+
+For converting TensorFlow models to CoreML format, the recommended approach is to use `coremltools 4.0` python package.
+
+Please see the  coremltools documentation on [Tensorflow conversion](https://coremltools.readme.io/docs/tensorflow-conversion) for example usage.
+
+To get the latest version of coremltools:
 
 ```shell
-pip install --upgrade tfcoreml
+pip install coremltools==4.0b1
 ```
 
-For the latest changes please see the [release
-notes](https://github.com/tf-coreml/tf-coreml/releases).
+For the latest changes in coremltools, please see the [release notes](https://github.com/apple/coremltools/releases/).
 
-Usage
-------
+tfcoreml 
+---------
 
-Please see the Tensorflow conversion section in the [Neural network
-guide](https://github.com/apple/coremltools/blob/master/examples/NeuralNetworkGuide.md)
-on how to use the converter. 
+`tfcoreml` package is **no longer maintained**. It can be used only if the **primary** deployment target is `iOS 12` or earlier. To install `tfcoreml`, please run:
```

Comment:
Here we should also point to documentation on how to use tfcoreml (if applicable)

- can point to https://github.com/apple/coremltools/blob/3.4/examples/NeuralNetworkGuide.md#tensorflow-1-converter, with condition that only use `tfcoreml.convert` if `minimum_ios_deployment_target` is `12` or `11.2`

- mention clearly that `tfcoreml.convert` only supports Tensforflow 1 , frozen protobuf format (".pb") which is a serialized form of frozen `graph_def` object

- If you want to convert a `tf.keras` model (from TF1 or TF2) then, follow these steps to save it in the .pb format and then use tfcoreml (you can add this detail in the next PR as well, or add it in the coremltools documentation and link to it from here)

Reply:
Done. Skipping point 1 as that link is confusing since it talks about both TF 1 other formats and deployment target 13 and all. Users with less context might find it contradictory and confusing. 

Point 3 can be covered once we provide such a utility function.